cashu/wallet/api/router.py

```diff
@@ -127,6 +128,46 @@ async def invoice(
     return
 
 
+@router.post(
+    "/swap",
+    name="Multi-mint swaps",
+    summary="Swap funds between mints.",
+    response_model=SwapResponse,
+)
+async def swap(
+    amount: int = Query(default=..., description="Amount to swap between mints"),
+    outgoing_mint: str = Query(default=..., description="URL of outgoing mint"),
+    incoming_mint: str = Query(default=..., description="URL of incoming mint"),
+):
+    if not settings.lightning:
+        raise Exception("this mint does not support lightning.")
```

Comment:
This setting applies to the local wallet/mint, but for a swap it only matters whether the incoming mint supports Lightning? Which I think you can only check if it does by requesting a mint invoice and seeing if it responds with one or an error, as this isn't specified in `/info`.

Could then avoid re-loading the incoming mint on L159 
```python
    # mint token in incoming mint
    wallet = await load_mint(wallet, incoming_mint)
```